### PR TITLE
Add search engine mood test and remove banner test

### PR DESCRIPTION
**Pull request by NohYujeong**

### CODE REVIEW

File: app/components/paperShow/refCitedTab/refCitedTab.scss [hunk 1]
```diff
@@ -13,7 +13,7 @@
     background-color: white;
     border-bottom: 1px solid $gray8;
     margin: 0;
-    z-index: 101;
+    z-index: 99;
```
Comment (TylorShin) [inline app/components/paperShow/refCitedTab/refCitedTab.scss]: is it okay with paper show's elements? (before / after scroll)

File: app/components/preNoted/suddenAlert.scss [hunk 2]
```diff
@@ -5,6 +5,7 @@
   right: 0;
   width: 371px;
   background-color: $dark-blue0;
+  z-index: 999;
```
Comment (TylorShin) [inline app/components/preNoted/suddenAlert.scss]: is it okay with paper show's elements? (before / after scroll)

File: app/components/paperShow/refCitedTab/refCitedTab.scss (hunk 1 above)
Comment (NohYujeong) [reply app/components/paperShow/refCitedTab/refCitedTab.scss]: Yes, I checked it and it worked without problems.
refCitedTab stick -> ok
ref / cited paper layout -> ok
pdf viewer -> ok

File: app/components/preNoted/suddenAlert.scss (hunk 2 above)
Comment (NohYujeong) [reply app/components/preNoted/suddenAlert.scss]: Yes, I checked it and it worked without problems.
refCitedTab stick -> ok
ref / cited paper layout -> ok
pdf viewer -> ok

**Comment by TylorShin** [comment]
👍

**Pull request merged**